### ISSUE TITLE
cucumber-language-server fails to start up

### ISSUE THREAD

**Issue by cbochs** [cucumber/language-server#17]
**Describe the bug**

I am attempting to get the language-server setup for use in neovim. However, it fails to start up with the following error.

```
❯ cucumber-language-server
file:///usr/local/lib/node_modules/@cucumber/language-server/bin/cucumber-language-server.js:2
require('source-map-support').install()
^

ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/usr/local/lib/node_modules/@cucumber/language-server/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at file:///usr/local/lib/node_modules/@cucumber/language-server/bin/cucumber-language-server.js:2:1
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)

Node.js v17.0.1
```

This is being run on the (most recent?) released version.

```
❯ bat package.json | grep version
  "version": "0.3.1",
```

The package was installed via npm.

```
❯ npm install -g @cucumber/language-server
```

**To Reproduce**
1. Run the server (i.e. `cucumber-language-server`)

**Expected behavior**
The server should start up without errors.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: macOS Monterey

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A

**Comment by smilovanovic** [comment]
+1 on this, same issue on my end

**Comment by aslakhellesoy** [comment]
The Cucumber language server isn't really ready for general use yet. There are still some issues to iron out.

I'm probably not going to have much time to work on it for the next few months, but I'd be happy to onboard anybody who's interested in contributing to it.

**Comment by aslakhellesoy** [comment]
@cbochs @smilovanovic this should be fixed in 0.3.3
